docs/api/webview-tag.md

```diff
@@ -545,9 +545,28 @@ Stops any `findInPage` request for the `webview` with the provided `action`.
 
 * `options` Object (optional)
   * `silent` Boolean (optional) - Don't ask user for print settings. Default is `false`.
-  * `printBackground` Boolean (optional) - Also prints the background color and image of
+  * `printBackground` Boolean (optional) - Prints the background color and image of
     the web page. Default is `false`.
-  * `deviceName` String (optional) - Set the printer device name to use. Default is `''`.
+  * `deviceName` String (optional) - Set the printer device name to use. Must be the system-defined name and not the 'friendly' name, e.g 'Brother_QL_820NWB' and not 'Brother QL-820NWB'.
+  * `color` Boolean (optional) - Set whether the printed web page will be in color or grayscale. Default is `true`.
```

Comment:
Is documenting these other parameters intentional as part of this PR ?

Reply:
yes, see PR body above:
> Also fixes missing documentation for <webview>.print().

Reply:
Sorry missed that, its better to pull that into separate PR. The release notes and PR title mislead.

Reply:
Since docs are using facing change, its better to do it in a separate PR for visibility. Can you please make this change. Thanks!

Reply:
will do!

Reply:
done!